client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -45,29 +112,23 @@ const CancelPurchaseRefundInformation = ( {
 
 	if ( isRefundable( purchase ) ) {
 		if ( isDomainRegistration( purchase ) ) {
-			text = i18n.translate(
-				'When you cancel your domain within %(refundPeriodInDays)d days of purchasing, ' +
-					"you'll receive a refund and it will be removed from your site immediately.",
-				{
-					args: { refundPeriodInDays },
-				}
+			text.push(
+				translate(
+					'When you cancel your domain within %(refundPeriodInDays)d days of purchasing, ' +
+						"you'll receive a refund and it will be removed from your site immediately.",
+					{
+						args: { refundPeriodInDays },
+					}
+				)
 			);
 		}
 
 		if ( isSubscription( purchase ) ) {
-			text = [
-				i18n.translate(
-					"We're sorry to hear the %(productName)s plan didn't fit your current needs, but thank you for giving it a try.",
-					{
-						args: {
-							productName: getName( purchase ),
-						},
-					}
-				),
-			];
+			text = [];
+
 			if ( includedDomainPurchase && isDomainMapping( includedDomainPurchase ) ) {
 				text.push(
-					i18n.translate(
```

Comment:
Thank you for changing these! That will definitely help the future of this file.